installation-linux/README.md

```diff
@@ -34,7 +34,7 @@ If not, run `micromamba activate autodock`
 micromamba install python=3.11 -y
 micromamba install pymol-open-source -y
 micromamba install rdkit numpy scipy -y
-micromamba install chemicalite matplotlib=3.7 pandas -y
+micromamba install chemicalite matplotlib=3.7 pandas multiprocess -y
```

Comment:
Although README says only MacOS needs multiprocess, the Google Linux backends seem to need multiprocess.